data.txt

```diff
@@ -0,0 +1 @@
+{'All': {'confirmed': 2804743, 'recovered': 207211, 'deaths': 67566, 'country': 'France', 'population': 64979548, 'sq_km_area': 551500, 'life_expectancy': '78.8', 'elevation_in_meters': 375, 'continent': 'Europe', 'abbreviation': 'FR', 'location': 'Western Europe', 'iso': 250, 'capital_city': 'Paris', 'lat': '46.2276', 'long': '2.2137', 'updated': '2021/01/09 07:22:49+00'}, 'French Guiana': {'lat': '4', 'long': '-53', 'confirmed': 13825, 'recovered': 9995, 'deaths': 73, 'updated': '2021/01/09 07:22:49+00'}, 'French Polynesia': {'lat': '-17.6797', 'long': '-149.4068', 'confirmed': 17241, 'recovered': 4842, 'deaths': 122, 'updated': '2021/01/09 07:22:49+00'}, 'Guadeloupe': {'lat': '16.265', 'long': '-61.551', 'confirmed': 8702, 'recovered': 2242, 'deaths': 155, 'updated': '2021/01/09 07:22:49+00'}, 'Martinique': {'lat': '14.6415', 'long': '-61.0242', 'confirmed': 6117, 'recovered': 98, 'deaths': 43, 'updated': '2021/01/09 07:22:49+00'}, 'Mayotte': {'lat': '-12.8275', 'long': '45.166244', 'confirmed': 6045, 'recovered': 2964, 'deaths': 56, 'updated': '2021/01/09 07:22:49+00'}, 'New Caledonia': {'lat': '-20.904305', 'long': '165.618042', 'confirmed': 40, 'recovered': 40, 'deaths': 0, 'updated': '2021/01/09 07:22:49+00'}, 'Reunion': {'lat': '-21.1151', 'long': '55.5364', 'confirmed': 9173, 'recovered': 8830, 'deaths': 42, 'updated': '2021/01/09 07:22:49+00'}, 'Saint Barthelemy': {'lat': '17.9', 'long': '-62.8333', 'confirmed': 206, 'recovered': 172, 'deaths': 1, 'updated': '2021/01/09 07:22:49+00'}, 'Saint Pierre and Miquelon': {'lat': '46.8852', 'long': '-56.3159', 'confirmed': 16, 'recovered': 15, 'deaths': 0, 'updated': '2021/01/09 07:22:49+00'}, 'St Martin': {'lat': '18.0708', 'long': '-63.0501', 'confirmed': 1025, 'recovered': 855, 'deaths': 12, 'updated': '2021/01/09 07:22:49+00'}, 'Wallis and Futuna': {'lat': '-14.2938', 'long': '-178.1165', 'confirmed': 4, 'recovered': 1, 'deaths': 0, 'updated': '2021/01/09 07:22:49+00'}}
```

Comment:
did you know [gitignore](https://www.atlassian.com/git/tutorials/saving-changes/gitignore) file?

Actually, we won't need a cache file in the repository code. Each "installation" of a script would create its own `data.txt`